<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateChild</class>
 <widget class="QMdiSubWindow" name="CreateChild">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMdiSubWindow{
background-image: url(:/res/images/bg2.jpg);
}
</string>
  </property>
  <widget class="QLabel" name="QuestionLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic Medium</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#1d1d1d;&quot;&gt;Type your question here :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="questionedit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Input Question Here</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Poll1Label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic Medium</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Enter poll option here :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="poll1edit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>270</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Poll Option #1</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="poll2edit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>370</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Poll Option #2</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Poll2Label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic Medium</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Enter poll option here :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="poll3edit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>470</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Poll Option #3</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Poll3Label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>470</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic Medium</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Enter poll option here :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submitbutton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>590</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Poll</string>
   </property>
   <property name="icon">
    <iconset theme="round">
     <normaloff>.</normaloff>.</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="header">
   <property name="geometry">
    <rect>
     <x>336</x>
     <y>32</y>
     <width>531</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:48pt; font-weight:600; text-decoration: underline; color:#000000;&quot;&gt;Create A Survey&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returntomenu">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>30</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Return to Menu</string>
   </property>
   <property name="icon">
    <iconset theme="round">
     <normaloff>.</normaloff>.</iconset>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QMdiSubWindow</class>
   <extends>QWidget</extends>
   <header>qmdisubwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
